documentation/developer_guide/installation.rst

```diff
@@ -238,34 +238,44 @@ You also need to tell cau-client where the regional CAU and the leader agent CAU
 Use of the SLA Management component
 -----------------------------------
 
-The SLA Management is a lightweight implementation of an SLA system, inspired by the WS-Agreement standard. It features (i) a REST interface to manage agreements, (ii) a background agreement assessment. 
+The SLA Management is a lightweight implementation of an SLA system, inspired by the WS-Agreement standard. It features (i) a REST interface to manage agreements, (ii) a background agreement assessment.
 
-An agreement is represented by a simple JSON structure. An example is shown below:
+To make use of the SLA Management on IT-1, you must install an agreement in the system. This agreement will be detected by the GUI when creating a service instance and will be associated to it.
+
+An agreement is represented by a simple JSON structure. Below is the default agreement that you should install. This agreement will check that the service operations are executed in less than one second. Modify the constraint to allow different time threshold. You must also modify the name of the agreement (including the details part) to match the name of a installed service. Install as many agreements as service kinds you want to observe.
 
 .. code-block:: bash
 
  {
-    "name": "an-agreement-name",
+    "name": "your-service-name-here",
```

Comment:
This is simple workaround to make an agreement specific to a service (only for IT-1!). If the name is arbitrary, it will be used as the default agreement. @FranCarpio : can you confirm this?